SGO/sgo/entities/Tag.ttl

```diff
@@ -0,0 +1,27 @@
+@prefix ogit:                   <http://www.purl.org/ogit/> .
+@prefix rdfs:                   <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix dcterms:                <http://purl.org/dc/terms/> .
+
+ogit:Tag
+	a rdfs:Class;
+	rdfs:subClassOf ogit:Entity;
+	rdfs:label "Tag";
+	dcterms:description "An Tag is a customer defined tag to help filter data";
+	# For ranges, see http://dublincore.org/documents/dcmi-period/
+	dcterms:valid "start=2020-08-04;";
+	dcterms:creator "Ben Neal";
+	ogit:scope "SGO";
+	ogit:parent ogit:Node;
+	ogit:mandatory-attributes (
+		ogit:name
```

Comment:
If it's really just the name, nothing would keep me from adding many tags with same name here, right?

Reply:
Name shouldn't be unique as can be reused across instances, and even re-used on the same instance. I don't see a use-case where we'd need the name to be unique.
I have also added an option attribute `color`